x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.test.ts

```diff
@@ -1202,6 +1222,26 @@ describe('EPM template', () => {
             },
           },
         },
+        {
+          'group.*.network': {
+            path_match: 'group.*.network',
+            match_mapping_type: 'object',
+            mapping: {
+              type: 'object',
+              dynamic: true,
+            },
+          },
+        },
+        {
+          'group.*': {
+            path_match: 'group.*',
+            match_mapping_type: 'object',
+            mapping: {
+              type: 'object',
+              dynamic: true,
+            },
+          },
+        },
```

Comment:
Example of intermediary objects created using dynamic mappings, as they contain dynamic parts.